pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -5706,10 +5706,10 @@ statfunc u32 cgroup_skb_generic(struct __sk_buff *ctx)
         return 1;
     }
 
-    int zero = 0;
-    net_event_context_t *neteventctx = bpf_map_lookup_elem(&cgroup_skb_events_scratch_map, &zero);
-    if (unlikely(neteventctx == NULL))
-        return 0;
+    // TODO: We may run into stack limit issue here.
```

Comment:
You can simply use the mechanism that we introduced with selecting a scratch buffer from the scratch map. As pointed out in the original PR, since `event_context_t` is the first field in `net_event_context_t`, it should just work :tm: .